### PR TITLE
fix space in folder name and add tests

### PR DESCRIPTION
**Pull request by sowu880**
Fix bug : folder contains space.

### CODE REVIEW

File: src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Client/OrasClientTests.cs [hunk 1]
```diff
@@ -47,10 +56,27 @@ private void PushOneLayerImage()
 
         private void PushMultiLayersImage()
         {
-            string command = $"push {_testOneLayerImageReference} {_baseLayerTemplatePath} {_userLayerTemplatePath}";
+            string command = $"push {_testMultiLayersImageReference} {_baseLayerTemplatePath} {_userLayerTemplatePath}";
             OrasExecution(command);
         }
 
+        [Theory]
```
Comment (ACMoretxj) [inline src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Client/OrasClientTests.cs:63]: Do we have test cases for the invalid output folder while pulling? Say `GivenInvalidOutputFolder_WhenPull_ExceptionWillBeThrown`.

File: src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public OrasClient(string imageReference)
 
         public async Task PullImageAsync(string outputFolder)
         {
-            string command = $"pull  {_imageReference} -o {outputFolder}";
+            string command = $"pull  {_imageReference} -o \"{outputFolder}\"";
```
Comment (yankunhuang-pku) [inline src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs:31]: Is it safer to add "" to the image reference?

File: src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Client/OrasClientTests.cs (hunk 1 above)
Comment (sowu880) [reply src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Client/OrasClientTests.cs:68]: Thanks, add it.

File: src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs (hunk 2 above)
Comment (sowu880) [reply src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs]: Thanks, fixed it.

**Review by yankunhuang-pku — APPROVED** [review]

**Review by ACMoretxj — APPROVED** [review]

**Review by moria97 — APPROVED** [review]

**Pull request merged**